app/Console/Commands/FilesystemSetupCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use Bouncer;
+use Illuminate\Console\Command;
+use Kanvas\Apps\Repositories\AppsRepository;
+
+class FilesystemSetupCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'filesystem:disk';
+
+    /**
+     * The console command description.
+     *
+     * @var string|null
+     */
+    protected $description = 'Command that setup the filesystem disk';
+
+    public function handle(): void
+    {
+        $app = AppsRepository::findFirstByKey(env('KANVAS_APP_ID'));
+
+        $app->set('filesystem-service', 's3');
```

Comment:
@arfenis same thing for this params , allow them to be pass as prompt like https://github.com/bakaphp/kanvas-ecosystem-api/blob/development/app/Console/Commands/KanvasAppSetupCommand.php#L35